docs/example-scenario/apps/fully-managed-secure-apps.md

```diff
@@ -0,0 +1,192 @@
+3. User requests flow through the ILB of the ASE, which in turn routes the traffic to the Contoso Expenses Web App.
+4. User then proceeds to create an Expense Report
+5. As part of the expense creation, the deployed API App is invoked to retrieve user's manager name and email
+6. The Created Expense Report is stored in Azure SQL Database
+7. To facilitate continuous deployments, code gets checked into the Azure DevOps instance
+8. The build VM has the Azure DevOps Agent installed which allows the build VM to pull the bits for the Web App to deploy to the ASE (as the Build VM is deployed in a Subnet inside the same VNET)
+
+### Components
+
+* The [App Service Environment][intro-to-app-svc-env] provides a fully isolated and dedicated environment for securely running the application at high scale. In addition, since ASE and the workloads that run on it are behind a VNET, this provides an additional layer of security and isolation. The requirement of high scale and isolation drove towards selecting ILB ASE.
+* This workload is using the [isolated App Service pricing tier][isolated-tier-pricing-and-ase-pricing] so the application is running in a private, dedicated environment in an Azure datacenter using Dv2-series VMs with faster processors, SSD storage, and double the memory-to-core ratio compared to Standard
+* Azure App Services [Web App][docs-webapps] and [API App][docs-apiapps] hosts web applications and RESTful APIs. These are hosted on the Isolated Pricing Tier plan which also offers auto-scaling, custom domains and so on, but in a dedicated tier.
+* Azure [Application Gateway][docs-appgw] is a web traffic load balancer oerating at Layer 7 that manages traffic to the web application. It offers SSL off-loading, that removes additional overhead on the web servers hosting the web app to decrypt traffic again.  
+* [Web Application Firewall][docs-waf] Web application firewall (WAF) is a feature of Application Gateway. The reason for enabling the WAF in the Application Gateway is to further enhance security. The WAF uses OWASP rules to further protect the web application against attacks such as cross-site scripting, session hijacks, and SQL injection.
+* [Azure SQL Database][docs-sql-database] Azure SQL Database was selected as majority of data in this application is relational data, with some data as documents and BLOB. 
+* [Azure Networking] Azure provides a variety of networking capabilities in Azure and the the netowrks can further be peered with other VNETS in Azure or connectivity can be established with on-prem data centers via Express Route or Site to Site. In this case, the Microsoft.Sql service endpoint is enabled on the VNET, then the VNET is allowed access in the Firewall settings hosting the Azure SQL Database. This was done to ensure the data is flowing just between the Azure VNET and the SQL Database instance.
```

Comment:
Introduce service endpoints first before mentioning "Microsoft.Sql service endpoint"

Reply:
Typos: "and the the netowrks"

Reply:
on-prem: s/b "on-premises"